src/TmsIntegrationExample.Services/TmsGatewayApi/Configuration/TmsGatewayConfiguration.cs

```diff
@@ -0,0 +1,24 @@
+﻿// <copyright file="TmsGatewayConfiguration.cs" company="Engaged Technologies">
+// Copyright (c) Engaged Technologies. All rights reserved.
+// </copyright>
+// <summary>
+//     Configuration class for TMS Gateway API
+// </summary>
+
+namespace TmsIntegrationExample.Services.TmsGatewayApi.Configuration
+{
+    public class TmsGatewayConfiguration
+    {
+        public System.Uri BaseAddress { get; set; }
+        public System.Uri LoginBaseAddress { get; set; }
+        public string LoginSubPath { get; set; }
+        public string AuthorizeSubPath { get; set; }
+        public string TokenSubPath { get; set; }
```

Comment:
In a real world scenario, the openid connect related endpoints should be retrieved from the discovery doc at `/.well-known/openid-configuration`. This is important because if we need to change where our identity provider is, it will ease the pain for any clients.

Reply:
not sure what we need here.

Reply:
We need to remove both of these, they don't need to be configured because we'll be calling out to the OpenIdProvider to get the discovery doc (always located at `/.well-known/openid-configuration`) to lookup the proper endpoints for the authorize and token requests.